README.md

```diff
@@ -125,6 +125,8 @@ We test supporting the brand-new Graph Warp Module (GWM) [18]-attached models fo
 - RSGCN ('rsgcn_gwm')
 - GIN ('gin_gwm')
 
+In the directory (examples/molnet_wle), we have implemented the newly proposed ``Weisfeiler-Lehman Embedding for Molecular Graph Neural Networks'' [26] for several GNN architectures. Please find the Readme in that directory for the usage and the details. 
```

Comment:
Thank you for adding term but sorry could you remove "the newly proposed", it will be not "new" in the future.